src/main/java/org/redquark/algorithms/datastructures/lists/CircularLinkedList.java

```diff
@@ -95,6 +95,10 @@ public void addAfterNode(T element, T elementInTheList) {
         }
         // Create a new node with the given data
         ListNode<T> newNode = new ListNode<>(element);
+        if(temp == tail) {
```

Comment:
Can we add proper spacing between `if` and `(`

Reply:
Updated the PR!